.gitignore

```diff
@@ -11,7 +11,6 @@ ssl/
 .DS_Store
 
 # Autogenerated directories
-bin/
 include/
```

Comment:
What are `include/` and `lib/` doing here. Any ideas @dittonjs?

Reply:
I removed those too in https://github.com/hypothesis/lms/pull/41